samples/meetings-live-caption/nodejs/index.js

```diff
@@ -0,0 +1,73 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// index.js is used to setup and configure your tab app
+
+// Import required packages
+const path = require('path');
+
+const express = require('express');
+const cors = require('cors');
+const MeetingApiHelper = require('./helpers/meetingApiHelper');
```

Comment:
Please check casing.

Reply:
It is defined as class so used pascal case